gammapy/irf/effective_area.py

```diff
@@ -491,3 +191,53 @@ def peek(self, figsize=(15, 5)):
         self.plot_energy_dependence(ax=axes[0])
         self.plot_offset_dependence(ax=axes[1])
         plt.tight_layout()
+
+    @classmethod
+    def from_parametrization(cls, energy_axis_true=None, instrument="HESS"):
```

Comment:
Since this is now a 2D table, one might expect to get correct offset response. Maybe this should not be exposed any more. (After all this is mostly for testing right?)

Reply:
Yes, this is only used for testing. I think adding a comment that the offset dependence is ignored is sufficient? In general I think such toy-models might be useful in certain situations (mostly testing). In combination with something like `EnergyDispersion2D.from_gauss()` and `EnergyDependentMultiGaussPSF` one can generate a full toy response..